google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -273,6 +274,10 @@ def submit(
         if network:
             self._gca_resource.network = network
 
+        # Prevents logs from being supressed on TFX pipelines
+        if self._gca_resource.pipeline_spec["sdkVersion"].startswith("tfx"):
```

Comment:
Please update the unit tests to include a mock tfx pipeline similar to this:
https://github.com/googleapis/python-aiplatform/blob/main/tests/unit/aiplatform/test_pipeline_jobs.py#L76

It can then be added to parameterize the current tests here:
https://github.com/googleapis/python-aiplatform/blob/main/tests/unit/aiplatform/test_pipeline_jobs.py#L240

You may need to update the expected call in the test to add `sdkVersion`: https://github.com/googleapis/python-aiplatform/blob/main/tests/unit/aiplatform/test_pipeline_jobs.py#L285